tests/fixtures/scenarios/exitCode/atmos.yaml

```diff
@@ -0,0 +1,21 @@
+base_path: "./"
+
+components:
+  terraform:
+    base_path: "components/terraform"
```

Comment:
```suggestion
    base_path: "../../components/terraform"
```

Reply:
Not sure why this?
The test would stop working because the directory is not present.

Reply:
But it is present, it's how our other scenarios are set up. We want a component library that can be reused across tests. There's another comment to move the exit-zero component to the component library. 

Look at other scenarios for examples.

Reply:
Thanks, this gave me perspective. Done